js/private/translate_package_lock.bzl

```diff
@@ -140,13 +155,25 @@ def npm_repositories():
         if "requires" in dep.keys():
             for n in dep["requires"].keys():
                 deps.append("@" + _repo_name(n, [d["version"] for (p, d) in lockfile["dependencies"].items() if p == n][0]))
+
+        name_with_version = _repo_name_and_version(name, dep["version"])
+
+        patches = patches_map.get(name, [])[:]
+        patches.extend(patches_map.get(name_with_version, []))
+
+        patch_args = patch_args_map.get(name, [])
```

Comment:
nit: I would make this symmetrical here with
```
patch_args.extend(patch_args.get(name_with_version, []))
```
Not likely to be used but it keeps the semantics for the two dicts the same

Reply:
Updated